res/TensorFlowLiteRecipes/SVDF_000/test.recipe

```diff
@@ -0,0 +1,61 @@
+operand {
+  name: "ifm"
+  type: FLOAT32
+  shape { dim: 1 dim: 16 }
+}
+operand {
+  name: "weight_feature"
+  type: FLOAT32
+  shape { dim: 64 dim: 16 }
+  filler {
+    tag: "gaussian"
+    arg: "0.0"
+    arg: "1.0"
+  }
+}
+operand {
+  name: "weight_time"
+  type: FLOAT32
+  shape { dim: 64 dim: 8 }
+  filler {
+    tag: "gaussian"
+    arg: "0.0"
+    arg: "1.0"
+  }
+}
+operand {
+  name: "bias"
+  type: FLOAT32
+  shape { dim: 64 }
+  filler {
+    tag: "gaussian"
+    arg: "0.0"
+    arg: "1.0"
+  }
+}
+operand {
+  name: "input_activation_state"
+  type: FLOAT32
+  shape { dim: 1 dim: 512 }
+}
+operand {
+  name: "ofm"
+  type: FLOAT32
+  shape { dim: 1 dim: 64 }
+}
+operation {
+  type: "SVDF"
+  svdf_options {
+    rank: 1
+    activation: RELU
+    asymmetric_quantize_inputs: false
+  }
+  input: "ifm"
+  input: "weight_feature"
+  input: "weight_time"
+  input: "bias"
+  input: "input_activation_state"
+  output: "ofm"
+}
+input: "ifm"
+output: "ofm"
```

Comment:
Q. Did you check this tflrecipe produce correct `SVDF` model.

Reply:
There are some difficulties with the last input for this SVDF op. How I mentioned before SVDF has one specific input tensor, that doesn't store any data (its buffer is empty), but is used in the kernel (as a scratch tensor) and stores quantization parameters, type and shape. And that is why I am not sure that everything is fine with this part of SVDF op. 
This recipe can produce svdf model and this model can be processed be luci-interpreter. 
If someone can tell me how i can check the correctness of this svdf model, I will be very grateful.

Reply:
I built this draft locally, converted this recipe to tflite using:

```
$ ./build/compiler/tflchef/tools/file/tflchef-file res/TensorFlowLiteRecipes/SVDF_000/test.recipe model.tflite
```

Then, I tried to infer this model using following script:

<details>
<summary> invoke_tflite.py </summary>

```python
import tensorflow as tf
import numpy as np

model_path = "./model.tflite"

with open(model_path, 'rb') as binf:
    interp = tf.lite.Interpreter(model_content=binf.read())

interp.allocate_tensors()

for idx, detail in enumerate(interp.get_input_details()):
    tensor_idx = detail['index']
    # generate random data
    tensor_data = np.random.randn(*detail['shape']).astype(detail['dtype'])
    # dump to file (if needed)
    np.save("input" + str(idx) + ".npy", tensor_data)
    # set data to interpreter
    interp.set_tensor(tensor_idx, tensor_data)

# execute model
interp.invoke()

# collect outputs to list
ouptuts = []
for idx, detail in enumerate(interp.get_output_details()):
    tensor_idx = detail['index']
    tensor_data = interp.get_tensor(tensor_idx)
    ouptuts.append(tensor_data)
    # dump to file (if needed)
    np.save("output" + str(idx) + ".npy", tensor_data)
```

</details>

But I got segfault (sorry for Russian error msg):

```
$ python3 invoke_tflite.py 
Ошибка сегментирования (стек памяти сброшен на диск)
```

My env is following:

```
$ python3
Python 3.6.9 (default, Oct  8 2020, 12:12:24) 
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
>>> tf.__version__
'2.2.0'
```

@BalyshevArtem Could you please take a look? Did I miss something? May be I need to use another tflite interpreter or version of tf?

Reply:
Please, note I tried same trick but with *kws_cut* model from https://github.com/Samsung/ONE/issues/8203#issuecomment-1000912620 and with model  [cut_svdf.zip](https://github.com/Samsung/ONE/files/7794893/cut_svdf.zip) (I forgot where I found this single-op model). Execution ends successfully.

Reply:
@m-bronnikov, 
Thanks for the help.
As I understood the problem was in the incorrect generation of this SVDF operation, it was necessary to add the is_variable field in svdf test.recipe. I added it in the last commit. I checked your script (but I used  tensorflow 2.6.0) and no more errors are found.

Reply:
I checked too and no errors occured. I guess it works fine now. Thank you!